libsolidity/codegen/ir/IRLValue.cpp

```diff
@@ -166,3 +166,79 @@ string IRStorageArrayLength::setToZero() const
 {
 	return storeValue("0", *TypeProvider::uint256());
 }
+
+IRMemoryItem::IRMemoryItem(
+	IRGenerationContext& _context,
+	std::string _address,
+	bool _byteArrayElement,
+	Type const& _type
+):
+	IRLValue(_context, &_type),
+	m_address(move(_address)),
+	m_byteArrayElement(_byteArrayElement)
+{ }
+
+string IRMemoryItem::retrieveValue() const
+{
+	if (m_byteArrayElement)
+		return "byte(0, mload(" + m_address + ")";
```

Comment:
For consistency, should this one also have `\n` at the end?

Reply:
Not only for consistency. It would straight out cause invalid code. Fixing.

Reply:
Are you sure? This is an expression and not a statement.

Reply:
Hmm you're right. Actually non of them should have `\n`